lib/organisms/gravity-form/gravity-form.directive.js

```diff
@@ -2,8 +2,18 @@ angular
   .module('lnPatterns')
   .directive('lnOGravityForm', lnOGravityForm);
 
+/**
+ * TODO:
+ *    - Expand the list of supported input types (see comments in form parser)
+ *
+ *    - NOTE: The validation feedback / update mechanism isn't very 'Angular'
+ *      at the moment
+ *
+ *    - Migrate to use of ng-model for form data gathering
+ */
+
 /*@ngInject*/
```

Comment:
I think we should use explicit injections instead of ngInject annotation, because this requires to setup ng-annotate on the app that uses the library to avoid problems with uglify.